packages/wallet/api/test/continuingInvitationExample.js

```diff
@@ -22,6 +22,7 @@ export const start = zcf => {
     updater.updateState('first offer made');
     return harden({
       uiNotifier: notifier,
+      publicNotifiers: { notifier },
```

Comment:
Part of the move away from `uiNotifier` is to name the notifiers for what they provide rather than the expected subscriber. We could demonstrate that more here with:
```suggestion
      publicNotifiers: { offers: notifier },
```

Reply:
Done